### PR TITLE
Enhance LDAP documentation

### PR DESCRIPTION
**Pull request by bellwood**
Incorporating @marvnrawley LDAP enhancements from #518

### CODE REVIEW

**Review by Carbenium — APPROVED** [review]
LDAP user here.

I can't comment on the Windows part since we use FreeIPA. The rest is a great explanation of the mappings.

**Comment by adamtbeames** [comment]
I'm still not able to get Windows LDAP working.
Some some pointers in the doc about how to debug would be very helpful.

File: docs/installation/ldap.md [hunk 1]
```diff
@@ -99,3 +103,17 @@ AUTH_LDAP_FIND_GROUP_PERMS = True
 AUTH_LDAP_CACHE_GROUPS = True
 AUTH_LDAP_GROUP_CACHE_TIMEOUT = 3600
 ```
+
+!!! info
+    "is_active" - you must map all users to at least this group if you want their account to be treated as enabled. Without this, your users cannot log in.
```
Comment (jeremystretch) [inline docs/installation/ldap.md]: Please capitalize the first word in each sentence for these descriptions

File: docs/installation/ldap.md [hunk 2]
```diff
@@ -99,3 +103,17 @@ AUTH_LDAP_FIND_GROUP_PERMS = True
...
+
+"is_staff" - users mapped to this group are enabled for access to the Administration tools; this is the equivalent of checking the "Staff status" box on a manually created user. This doesn't necessarily imply additional privileges, which still needed to be assigned via a group, or on a per-user basis.
+
+"is_superuser" - users mapped to this group in addition to the "is_staff" group will be assumed to have full permissions to all modules. Without also being mapped to "is_staff", this group observably has no impact to your effective permissions.
+
+!!! info
+    It is also possible map user attributes to Django attributes:
+AUTH_LDAP_USER_ATTR_MAP = {
```
Comment (jeremystretch) [inline docs/installation/ldap.md]: Probably want to wrap this in a code block and ensure proper indentation

**Review by jeremystretch — CHANGES_REQUESTED** [review]

**Comment by jeremystretch** [comment]
Pinging @bellwood - Are you available to make these changes?

**Comment by bellwood** [comment]
Been burried lately - will attempt to get a revised PR this week

**Comment by bellwood** [comment]
Sorry for the late edits on this - been all over the place lately

Thanks for the feedback =)

**Pull request merged**